src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/Models/ExtendedCodeActionContext.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using OmniSharp.Extensions.LanguageServer.Protocol.Models;
+using OmniSharp.Extensions.LanguageServer.Protocol.Serialization;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.CodeActions.Models
+{
+    class ExtendedCodeActionContext : CodeActionContext
```

Comment:
```suggestion
    internal class ExtendedCodeActionContext : CodeActionContext
```

Reply:
I thought the analyzers we added should have complained about this, maybe they got turned off or the settings altered. Will have to look into that.